packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -239,7 +242,8 @@ export class IndexedDbPersistence implements Persistence {
     lruParams: LruParams,
     private readonly queue: AsyncQueue,
     serializer: JsonProtoSerializer,
-    private readonly sequenceNumberSyncer: SequenceNumberSyncer
+    private readonly sequenceNumberSyncer: SequenceNumberSyncer,
+    private readonly forceOwningTab: boolean
```

Comment:
Do you mind adding a short inline comment for this and `allowTabSynchronization`? The rest of the arguments seem obvious.

Reply:
done.